hook.js

```diff
@@ -62,17 +62,14 @@ function getConfig(repo, branch) {
     var configFilename = '.doiuse';
 
     repoClient.contents(configFilename, branch, function(error, configFileMetadata) {
-        var defaultConfig = ['last 2 versions'];
-        var configFileContent, config;
-
-        if (error) {
-            // The configuration file does not exist
-            config = defaultConfig;
-        } else {
-            configFileContent = new Buffer(configFileMetadata.content, 'base64').toString();
+        var config = ['last 2 versions'];
 
+        // If the file is found, massage the file contents
```

Comment:
Small nitpick. Maybe we can also mention that we'll use that configuration? Something like:

> If the file is found, massage the file contents **and use that configuration**